app/create.py

```diff
@@ -58,11 +58,13 @@ def post(self):
         else:
             if not self.params.given_name:
                 return self.error(400, _('Name is required.  Please go back and try again.'))
-        if not self.params.author_name:
-            if self.params.clone:
-                return self.error(400, _('The Original author\'s name is required.  Please go back and try again.'))
-            else:
-                return self.error(400, _('Your name is required in the "Source" section.  Please go back and try again.'))
+        
+        if not self.params.own_info:
```

Comment:
As discussed in the meeting, you'll have to fill in author_name automatically if inputting own info.
So maybe a better option is to do something like this above:

if self.params.own_info == "yes":
  self.params.author_name = self.params.given_name

..etc